notebooks/pandas.ipynb

```diff
@@ -0,0 +1,348 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"top\"></a>\n",
+    "<div style=\"width:1000 px\">\n",
+    "\n",
+    "<div style=\"float:right; width:98 px; height:98px;\">\n",
+    "<img src=\"https://raw.githubusercontent.com/Unidata/MetPy/master/src/metpy/plots/_static/unidata_150x150.png\" alt=\"Unidata Logo\" style=\"height: 98px;\">\n",
+    "</div>\n",
+    "\n",
+    "<h1>Pandas</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "---\n",
+    "<div style=\"float:right; width:250 px\"><img src=\"../instructors/images/pandas_timeseries_example.png\" alt=\"Timeseries of Temperature and Pressure from Aug 1st to 15th\" style=\"height: 300px;\"></div>\n",
+    "\n",
+    "\n",
+    "### Focuses\n",
+    "* Using this notebook we will become familiar with Pandas as a tool for quick and easy data analysis! [Pandas](https://pandas.pydata.org/docs/index.html)\n",
+    "* We will use pandas to organize, process, and plot miscellaneous data\n",
+    "* Learn basic subsetting, displaying, and arithmetic operations with pandas\n",
+    "* Plot data from a pandas DataFrame with [Matplotlib](https://matplotlib.org/)\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "### Objectives\n",
+    "1. [Create a Pandas DataFrame](#1.-Create-a-Pandas-DataFrame)\n",
+    "1. [Subsetting a DataFrame](#2.-Subsetting-a-DataFrame)\n",
+    "1. [Plot data from our DataFrame](#3.-Plot-data-from-our-DataFrame)\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports\n",
+    "We will begin by importing the important packages to be used throughout! Instructors, generally keep your imports to before your objectives unless there is an important takeaway from what you're importing or the way you're doing it. Give a brief description if necessary in more introductory notebooks.\n",
```

Comment:
Looks like a comment from the template got left in here - "Instructors, generally keep...."